test/acceptance/ApiSubscriptionFieldsSpec.scala

```diff
@@ -133,12 +132,12 @@ class ApiSubscriptionFieldsSpec extends AcceptanceTestSpec
       val sfrFieldsId = contentAsJson(resultFuture).validate[SubscriptionFieldsResponse]
 
       sfrFieldsId.isSuccess shouldBe true
-      sfrFieldsId.get should matchPattern { case SubscriptionFieldsResponse(FakeRawIdentifier, `fieldsId`, SampleFields1) => }
+      sfrFieldsId.get should matchPattern { case SubscriptionFieldsResponse(`fakeRawClientId`, "acontext", "1.0.2", `fieldsId`, SampleFields1) => }
     }
 
-    scenario("the API is called to GET with a known application identifier") {
+    scenario("the API is called to GET with a known client identifier") {
```

Comment:
`the API is called to GET existing subscription fields by application clientId`

Reply:
Done